resources/lang/en.json

```diff
@@ -311,6 +311,7 @@
   "manual:pictures:picture_allow_delete": "If enabled pictures can be deleted on result page directly after they have been taken.",
   "manual:pictures:picture_cheese_time": "Set a time to display \"Cheeeeeeeese!\" after the countdown.",
   "manual:pictures:picture_cntdwn_time": "Set your countdown time.",
+  "manual:pictures:picture_cntdwn_offset": "Set an offset to the contdown to compensate for cheese and or focus and shutter delay.",
```

Comment:
countdown not contdown

Reply:
Also `and / or`

Reply:
What about: "Set an offset to the countdown to compensate for cheese time, focus and shutter delay." ?

Reply:
`Set an offset to the countdown to compensate the delay because of cheese time, focus and shutter speed.`

Maybe?

Reply:
perfekt :)